modules/auxiliary/scanner/winrm/winrm_cmd.rb

```diff
@@ -38,7 +38,7 @@ def run_host(ip)
     streams = winrm_run_cmd(datastore['CMD'])
     return unless streams.class == Hash
     print_error streams['stderr'] unless streams['stderr'] == ''
-    print_good streams['stdout']
+    print_good "#{ip}:#{rport} => " + streams['stdout']
```

Comment:
`peer` may be registered here, try: `"#{peer} => "`.  If it doesn't work, thats fine.

Here you did string interpolation, then string concatenation.  I would recommend switching to just interpolation: `#{ip}:#{rport} => #{streams['stdout']}` (change out ip/rport if peer works)